libraries/streaming-e2e-tests/react-app/package.json

```diff
@@ -6,10 +6,11 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
-    "botframework-webchat": "4.10.1",
+    "botframework-streaming": "next",
+    "botframework-webchat": "latest",
```

Comment:
@joshgummersall I think we were on this version of Web Chat because there were some DOM/V-DOM changes being discussed in Web Chat. @Zerryth do you remember the specifics of why we were pinned to this version?